contracts/extensions/CoinMachine.sol

```diff
@@ -358,11 +358,11 @@ contract CoinMachine is ColonyExtension {
 
   /// @notice Get the maximum amount of tokens a user can purchase in total
   function getUserLimit(address _user) public view returns (uint256) {
-    // ((max(soldTotal, targetPerPeriod) * userLimitFraction) - soldUser) / (1 - userLimitFraction)
-    return (userLimitFraction == WAD || whitelist == address(0x0)) ? UINT256_MAX : wdiv(
-      sub(wmul(max(soldTotal, targetPerPeriod), userLimitFraction), soldUser[_user]),
-      sub(WAD, userLimitFraction)
-    );
+    return
+      (userLimitFraction == WAD || whitelist == address(0x0)) ?
+      UINT256_MAX :
+      sub(wmul(add(ERC20(token).balanceOf(address(this)), soldTotal), userLimitFraction), soldUser[_user])
```

Comment:
Can we use `getTokenBalance()` instead of making the direct call? For consistency mostly.